docs/android/submit.md

```diff
@@ -0,0 +1,73 @@
+
+### Checklist for Submitting a PR
+
+When creating a PR, GitHub pre-fills the description with a checklist. Ensure you follow all the steps. Here's an extended checklist to help you:
+
+- ✅ **PR Description**: Provide a clear and complete description of your changes.
+- ✅ **Tests**: Add all necessary tests following our [testing guidelines](testing/introduction).
+- ✅ **Documentation**: Ensure your code is properly documented.
+- ✅ **UI Changes**: Include screenshots if the UI is modified.
+- ✅ **User Documentation**: If user documentation needs updates, open a PR on [GitHub](https://github.com/home-assistant/companion.home-assistant).
+- ✅ **Developer Documentation**: If this documentation needs updates, open a PR on [GitHub](https://github.com/home-assistant/developers.home-assistant/).
+- ✅ **Builds**: Verify that everything builds properly (app, automotive, wear) locally.
+- ✅ **Best Practices**: Follow the [best practices](bestpractices).
+- ✅ **Code Style**: Adhere to the [code style](codestyle).
+- ✅ **Linting**: Ensure no lint issues are introduced ([linter](linter)).
+
+### Opening a Draft PR
+
+If your PR isn't ready for official review but you'd like feedback, open it in **draft mode**. This will still trigger the CI on every commit and is especially useful when working on CI-related changes.
+
+:::warning
+**Avoid unnecessary CI runs**  
+Running CI workflows consumes significant resources. If your work is incomplete, postpone opening the PR (even in draft mode) unless necessary. Let's be mindful of resource usage and our planet. 🌍 But it doesn't prevent you from pushing regularly to avoid loosing your work.
+:::
+
+### Review Process
+
+#### Who Can Review?
+
+Everyone can comment on your PR since it's public. We encourage contributions through reviews. Reviewing can be quicker than coding, and even a 10-minute review can be valuable.  
+
+If you're not confident in **coding**, you can still help by:  
+
+- Testing the feature by installing the APK (it's available on the Checks tab of the PT).  
+- Providing feedback on UI/UX.  
+- Reporting crashes or bugs.
+
+#### Getting Approval from a Maintainer
+
+Once your PR meets the checklist requirements, wait for a maintainer to review it. Remember, maintainers are volunteers contributing in their free time. Be respectful, patient, and kind.  
+
+Feedback from maintainers will come as:  
+
+- **Comments**: Suggestions or required changes in the code.  
+- **Questions**: Questions about how things are working.
+- **Suggestions**: Improvements or alternative approaches.
+
+### After Receiving Feedback
+
+#### Re-ask for Review
+
+If you've addressed feedback and pushed changes to your PR, you can request a re-review from maintainers. Ensure the CI is green before doing so.
```

Comment:
Doesn't the bot handle this?

Reply:
Only if reviewers click request changes. (I personally find the bot a bit agressive so don't use it in case of minor comments or if I haven't reviewed everything. If that's the case, you may need to re-request manually or wait.)

Reply:
Yes, but in core, if you as author click the ready for review button, the bot will ask the right people for a review, which is seen as less agressive than if you as author manually request reviews from people.

But I agree, with minor comments I rather fix them myself or just use the comment if they are not a blocker

Reply:
I don't think so on our repo yet. I'll look into it